core_lib_generator/file_generators/test_util_generator.py

```diff
@@ -0,0 +1,22 @@
+import os
+from core_lib.helpers.string import snake_to_camel
+from core_lib_generator.file_generators.template_generator import TemplateGenerator
+
+
+class UtilTestGenerateTemplate(TemplateGenerator):
+    def generate(self, template_content: str, yaml_data: dict, core_lib_name: str, file_name: str) -> str:
+        camel_case_name: str = snake_to_camel(core_lib_name)
+
+        import_instance_string: str = f'from {core_lib_name}_instance import {camel_case_name}'
+        template_content = template_content.replace('# template core_lib_instance', import_instance_string)
```

Comment:
can you verify if the util is using this import or it is needed here, I don't see this string the util file in the template